projects/packages/my-jetpack/src/products/class-crm.php

```diff
@@ -27,7 +27,10 @@ class Crm extends Product {
 	 *
 	 * @var string
 	 */
-	public static $plugin_filename = 'zero-bs-crm/ZeroBSCRM.php';
+	public static $plugin_filename = array(
+		'zero-bs-crm/ZeroBSCRM.php',
+		'crm/ZeroBSCRM.php',
```

Comment:
This is what the plugin slug seems to be during development. The My Jetpack link should show in production without it, but saw other plugins have their dev slugs in this array so it seemed best to add it